### PR TITLE
Jetpack Connect: Update plan subheading

### PR DESCRIPTION
**Pull request by tyxla**
#### Changes proposed in this Pull Request

* Jetpack Connect: Update plan headings/subheadings to align with the fact that we offer Backups

#### Preview

Before:

After:

#### Testing instructions

* Checkout this branch.
* Go to http://calypso.localhost:3000/jetpack/connect/store
* Verify you can see the new strings.

### CODE REVIEW

File: client/jetpack-connect/plans-grid.jsx [hunk 1]
```diff
@@ -42,12 +43,22 @@ class JetpackPlansGrid extends Component {
 	renderConnectHeader() {
 		const { isLanding, translate } = this.props;
 
-		const headerText = translate( 'Explore our Jetpack plans' );
+		const headerText = translate( 'Explore Jetpack' );
 		let subheaderText = translate( "Now that you're set up, pick a plan that fits your needs." );
+		if ( isEnabled( 'plans/jetpack-backup' ) ) {
+			subheaderText = translate(
+				"Now that you're set up, pick a solution or plan that fits your needs."
```
Comment (a8ci18n) [inline client/jetpack-connect/plans-grid.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Now that you're set up, pick a plan that fits your needs.' )` *ES Score: 7*

**Comment by michelleweber** [comment]
This feels a little confusing, because it now mentions solutions in two places -- as a reader, I don't know what "solutions" means in this context (isn't Jetpack itself a solution? Why are there other solutions? How many problems do I have?), and then I don't know which of the things to click on to get more information.

 I would:

- Make the first headline "Explore Jetpack plans"
- Remove "solutions" from the subhead there so it's "Pick a plan that fits your needs."
- Get rid of the "solutions" headline, because it's meaning is unclear (and presumably we'd prefer that people sign up for plans rather than just backups).
- Make the text there "Just need backups? Learn about backup-only options." Or if the word "solutions" is important to include, "Just need backups? Learn about backup-only solutions."

What do you think, @obi2020?

**Comment by tyxla** [comment]
Thanks a bunch @michelleweber!  I'm happy to do that, but that's contradicting to the initial suggestion by @p3ob7o. I'm not sure what's serving our purpose better, and I'm not a wordsmith myself, so I'll leave others discuss and decide what's best.

**Comment by robertbpugh** [comment]
@michelleweber I agree with your feedback in a vacuum, but this one is a bit more complicated. Here's a little more context:

We are introducing the ability to purchase single features by themselves, the first is Jetpack Backup. After much discussion of what to call these single features, we landed on "Solution". It's not super clear, but other words were also not clear.  Would also be nice to not name them anything, but some pages required a word or rewriting the UI which is out of scope.  We are also moving away from plans in the near future (some plans may end up staying, but there will be different types of individual features and bundles that will better match user needs - still TBD) so just using plans no longer works. So we're in the quite the pickle for this page.

**Comment by robertbpugh** [comment]
Also, we will be introducing other individual features like Jetpack Scan and Anti-spam, so we can't use Backups as the header for the single feature section.

**Comment by michelleweber** [comment]
Okay, that's super helpful. Given that, I'd say:
- I'd still remove "solutions" from the first subhead, because it confusing to see it in both places.
- I'd think use the second subhead to explain a little more. "Just need backups? Learn about single-issue solutions."

**Comment by crunnells** [comment]
I like Michelle's comments above, but thought I'd throw my two cents in (since "single-issue solution" sounds funny to me). Maybe:

> "Just need backups? Learn about add-on solutions."

The more I think about it, the more I wish we had a better word than "solutions."

**Comment by p3ob7o** [comment]
> I like Michelle's comments above, but thought I'd throw my two cents in (since "single-issue solution" sounds funny to me). Maybe:
> 
> > "Just need backups? Learn about add-on solutions."
> 
> The more I think about it, the more I wish we had a better word than "solutions."

To be clear, I don't feel attached to the term at all and I am happy to consider alternatives. I originally used "product" but it was brought to my attention that it could confuse users and ourselves. The essence is to differentiate a "plan" that is a collection of features across the board, with this new thing that is a smaller collection of features aiming at fulfilling ONE specific need.

**Comment by tyxla** [comment]
Thanks y'all for chiming in! 

I'm a bit confused as to what's the last copy we chose here. Just to clarify what I'm asking for: we have two headings and two subheadings - and from what y'all are suggesting, I guess we're going with:

---

**Explore our Jetpack plans**

Pick a plan that fits your needs.

**Solutions**

Just need backups? Learn about add-on solutions.

---

How does that sound @michelleweber @crunnells @robertbpugh @p3ob7o

**Comment by robertbpugh** [comment]
I started iterating on it, but I think what you have is fine for now,
Marin. We'll be updating this page within a few months for Scan so it gives
us more time to evaluate if Solutions is the best word.

**Comment by tyxla** [comment]
Alright, updated the copy, thanks folks! This is ready for a review.

**Pull request merged**